grr/core/grr_response_core/lib/casing.py

```diff
@@ -0,0 +1,73 @@
+# Lint as: python3
+"""Functions to convert strings between different case styles."""
+
+
+def SnakeToCamel(snake_str: str) -> str:
+  """Convert a snake_case string representing one identifier to lowerCamelCase.
+
+  The function uses a best-effort approach to convert the given string to a
+  valid lowerCamelCase string, meaning that it converts even strings that use
+  multiple consecutive underscores between words and/or that start/end with an
+  underscore.
+
+  Args:
+    snake_str: A snake_case string representing a single identifier.
+
+  Returns:
+    A lowerCamelCase representation of the given string.
+  """
+  # Extract the words from the snake_case string.
+  words = list(filter(lambda word: len(word) > 0, snake_str.split("_")))
+  if not words:
+    return ""
+
+  words[:] = map(str.lower, words)
+
+  lower_camel_str = words[0] + ''.join(map(str.capitalize, words[1:]))
+
+  return lower_camel_str
```

Comment:
`lower_camel_str` is kind of useless, why not just `return`?

Reply:
Thank you for your suggestion, I made the change.